sql/moz-fx-data-shared-prod/telemetry_dev_cycle_derived/firefox_major_release_dates_v1/query.sql

```diff
@@ -0,0 +1,17 @@
+WITH firefox_releases AS (
+  SELECT
+    CAST(REGEXP_EXTRACT(build.target.version, r"^\d+") AS INT64) AS version,
+    build.target.channel,
+    DATE(MIN(build.download.date)) AS publish_date
+  FROM
+    `mozdata.telemetry.buildhub2`
+  WHERE
+    build.target.channel IN ("release", "beta", "nightly")
+  GROUP BY
+    version,
+    channel
+)
+SELECT
```

Comment:
Seems like unnecessary stage here? We could just remove the CTE syntax and not have the SELECT * here?

Reply:
this is left over from my dbt days ... where you always do a select * from final at the end ... so much easier to debug ... and the data is small enough that it's not a problem to select it again?

Reply:
follow up ticket